File: C/max_consecutive_ones.c
int findMaxConsecutiveOnes(int* nums, int numsSize){
    int result = 0;
    int count = 0;
        
    for(int i = 0; i < numsSize; i++) {
        if(nums[i] == 1) count ++;
        if(nums[i] == 0 || i == numsSize-1) {
            if(count > result) result = count;
            count = 0;
        }
    }
            
    return result;
}

File: Python3/max_consecutive_ones.py
class Solution:
    def findMaxConsecutiveOnes(self, nums: List[int]) -> int:
        result = 0
        count = 0
        
        for i in range(len(nums)):
            if nums[i] == 1: 
                count += 1
            if nums[i] == 0 or i == len(nums)-1:
                if count > result:
                    result = count
                count = 0
        
        return result
